canonical-websites/maas.io
# Upgrade to latest VBT

## Done

- Bump to latest VBT
- Move closing div to fix layout issue
- Move external link class and remove span
- Temp fix for unescaped hashes in svgs
- Update .run script

## QA

- ./run serve
- Go to [http://0.0.0.0:8006](http://0.0.0.0:8006) and sure the ‘Automation’ section of 'Key features’ isn’t broken like this 
![screen shot 2017-07-25 at 16 13 49](https://user-images.githubusercontent.com/2152/28579363-6305dd4c-7154-11e7-9da7-8b56069526dd.png)

- Check that the ‘Docs’ external link isn’t broken like this 
![screen shot 2017-07-25 at 16 13 42](https://user-images.githubusercontent.com/2152/28579400-7cc9d2c4-7154-11e7-9ee8-626e781d8f24.png)

- Go to [http://0.0.0.0:8006/install](http://0.0.0.0:8006/install) and check that the you can see the code snippet icons, you can’t in this screenshot 
![screen shot 2017-07-25 at 16 14 03](https://user-images.githubusercontent.com/2152/28579509-ca6d2a58-7154-11e7-8dba-5188b766e59a.png)
